app/renderer/components/SelectBar.tsx

```diff
@@ -0,0 +1,79 @@
+import {Button, ButtonGroup} from "@blueprintjs/core"
+import {WorkingHoursAdvanced} from "app/types/settings"
+import React from "react"
+
+import {FormGroup} from "reactstrap"
```

Comment:
This should be done using existing blueprint components / custom markup, don't want to add a whole new dependency.